extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -238,8 +238,9 @@ async function getCommandsInPath(env: { [key: string]: string | undefined } = pr
 			const files = await vscode.workspace.fs.readDirectory(vscode.Uri.file(path));
 
 			for (const [file, fileType] of files) {
-				if (!labels.has(file) && fileType !== vscode.FileType.Unknown && fileType !== vscode.FileType.Directory && await isExecutable(path + pathSeparator + file)) {
-					executables.add({ label: file, path: path + pathSeparator + file });
+				const formattedPath = getFriendlyFolderPath(vscode.Uri.joinPath(vscode.Uri.from({ scheme: 'file', path }), file), pathSeparator);
```

Comment:
Can `vscode.Uri.file(path)` be put in a variable and also used below in the `joinPath`?